storage/src/cache/fscache/mod.rs

```diff
@@ -146,6 +149,31 @@ impl BlobCacheMgr for FsCacheMgr {
         self.get_or_create_cache_entry(blob_info)
             .map(|v| v as Arc<dyn BlobCache>)
     }
+
+    fn check_stat(&self) {
+        let guard = self.blobs.read().unwrap();
+        if guard.len() != self.blobs_need {
+            info!(
+                "blob mgr not ready to check stat, need blobs {} have blobs {}",
+                self.blobs_need,
+                guard.len()
+            );
+            return;
+        }
+
+        let mut all_ready = true;
+        for (_id, entry) in guard.iter() {
+            if !entry.is_all_data_ready() {
+                all_ready = false;
+                break;
+            }
+        }
+
+        if all_ready {
+            self.worker_mgr.stop();
+            self.metrics.data_all_ready.store(true, Ordering::Release);
```

Comment:
Can we also stop the periodic cache state checker?

Reply:
cause this is a gloable checker , if we want stop it , then need make sure all FsCacheMgr are "data ready".  And need restart it when we create a new FsCacheMgr, does this make sense ?

Reply:
Oops. You are right. We can't stop the timer